runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -260,21 +260,27 @@ public void onConfigChanged(ConfigChanged event)
 			switch (event.getKey())
 			{
 				case "warnForDefensive":
+					sendChatMessage("You have " + (enabled ? "enabled" : "disabled") + " the warning for " + returnAttackColor(Skill.DEFENCE) + " XP." );
```

Comment:
This should definitely not be an ingame chat message, and there is already a debug statement generated upon config changes.